app/Policies/PostPolicy.php

```diff
@@ -56,11 +55,11 @@ public function show($user, Post $post)
     /**
      * Determine if the given post can be updated by the user.
      *
-     * @param  Illuminate\Contracts\Auth\Authenticatable $user
+     * @param  $user
      * @param  Rogue\Models\Post $post
      * @return bool
      */
-    public function update(Authenticatable $user, Post $post)
+    public function update($user, Post $post)
     {
         return is_staff_user() || $user->northstar_id === $post->northstar_id;
```

Comment:
I think we'll want to check if user exists here before comparing IDs, and pull the `is_staff_user` up into a separate condition before that. Otherwise I think this might fail if `$user` is null.

```php
if (is_staff_user()) {
    return true;
}

return $user && $user->northstar_id === $post->northstar_id;
```

Reply:
Although this might not be a big deal since we have this endpoint protected by the `auth` middleware... still, can't hurt to be defensive about it!

Reply:
agreed - updated!